utils.py

```diff
@@ -0,0 +1,19 @@
+import logging.config
+import os
+import yaml
+
+
+def setup_logging(default_path='config/logging.yaml',
+                  default_level=logging.INFO,
+                  env_key='LOG_CFG'):
+
+    path = default_path
```

Comment:
Redundant.  Rename argument `default_path` to `path` and the assignment isn't needed.  Also rename `default_level` to `level`.